test/trento_web/controllers/v1/cluster_controller_test.exs

```diff
@@ -141,6 +142,117 @@ defmodule TrentoWeb.V1.ClusterControllerTest do
     end
   end
 
+  describe "request operation" do
+    test "should fallback to not found if the resource is not found", %{
+      conn: conn,
+      api_spec: api_spec
+    } do
+      conn
+      |> post("/api/v1/clusters/#{UUID.uuid4()}/operations/cluster_maintenance_change")
+      |> json_response(:not_found)
+      |> assert_schema("NotFound", api_spec)
+    end
+
+    test "should fallback to operation not found if the operation is not found", %{
+      conn: conn,
+      api_spec: api_spec
+    } do
+      %{id: cluster_id} = insert(:cluster)
+
+      conn
+      |> post("/api/v1/hosts/#{cluster_id}/operations/unknown")
+      |> json_response(:not_found)
+      |> assert_schema("NotFound", api_spec)
+    end
+
+    test "should respond with 500 if cluster maintenance change does not receive needed params",
```

Comment:
typo
```suggestion
    test "should respond with 422 if cluster maintenance change does not receive needed params",
```

Reply:
yes, copy/paste error. I will change the entry that had the initial error as well